Example/Emission/AppDelegate.m

```diff
@@ -119,18 +119,19 @@ - (void)setupEmissionWithUserID:(NSString *)userID accessToken:(NSString *)acces
   AppSetup *setup = [AppSetup ambientSetup];
 
   EmissionKeys *keys = [[EmissionKeys alloc] init];
-  
+
   AREmissionConfiguration *config = [[AREmissionConfiguration alloc] initWithUserID:userID
                                                                 authenticationToken:accessToken
                                                                           sentryDSN:nil
                                                                stripePublishableKey:[keys stripePublishableKey]
                                                                    googleMapsAPIKey:nil
+                                                                 mapBoxAPIClientKey:[keys mapBoxAPIClientKey]
```

Comment:
Outside the scope of this PR, I'd like us to move to passing in `keys` directly, instead of pulling everything out. If Emission defines a protocol that the object needs to conform to (one that matches `ArtsyKeys` in Eigen), then it should be straightforward.

Reply:
emission-nebula will need _some_ env car defined for the build to work (even if the value isn’t valid).

Reply:
https://travis-ci.org/artsy/emission-nebula/settings Added, just to be clear in cocoapods we have it beginning with a lowercase `m`, so I've added `mapBoxAPIClientKey`

Reply:
Thanks! If you look at the [Podfile](https://github.com/artsy/emission/blob/bb48bcfa3271e8ff10d67828e2a673215aef9530/Example/Podfile#L68-L70) that has the keys defined, the keys _always_ start with a capital, but get generated into a lower-case property during `pod install`. So you'll need `MapBoxAPIClientKey`, matching your change in the `Podfile`. Let me know if I can clarify 👍

Reply:
Thanks for the clarification, I added it with the uppercase M :)

Reply:
@ashfurrow I don't see env vars defined for other cocoapods keys in emission-nebula TravisCI settings. Am I looking in the wrong place.

Reply:
Looks like they're embedded in the `travis.yml`: https://github.com/artsy/emission-nebula/blob/master/.travis.yml#L25-L29

Both should work together